src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -224,6 +224,8 @@ public static class Captions
       public static readonly string SimulationWasCanceled = "Simulation was canceled";
       public static readonly string SelectMappingToShowObservedData = "Select mapping to show observed data";
       public static readonly string DoNotShowThisAgainUntilRestart = "Do not show this again until restart";
+      public static readonly string AddPoint = "Add Point";
+      public static string EditTableParameter(string parameter, bool editable) => $"{(editable ? "Edit" : "Show")} table parameter '{parameter}'";
```

Comment:
unusual to have a parmaeter dictating the caption as opposed to two constant but why nit